eng/common/TestResources/Remove-TestResources.ps1

```diff
@@ -116,6 +119,13 @@ if (!$ResourceGroupName) {
     $ResourceGroupName = "rg-$BaseName"
 }
 
+$root = [System.IO.Path]::Combine("$PSScriptRoot/../../../sdk", $ServiceDirectory) | Resolve-Path
```

Comment:
We should skip this step if the value of `ServiceDirectory` is an empty string otherwise we might run too many remove-test-resources-pre.ps1